ci-operator/step-registry/storage/destroy/csi-aws-efs/storage-destroy-csi-aws-efs-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: storage-destroy-csi-aws-efs
+  from_image:
+    namespace: ocp
```

Comment:
Good point, ok for me, thanks! In the step only need the aws and yq in the image.

Reply:
if use `from: upi-installer`, it would have to depend on the definition of `upi-installer` base image in ci yaml file, while using a fixed version of image would does not have that dependency.

Reply:
Good to know, thanks! If so I prefer to use a fixed version to avoid the extra depend base image definition.